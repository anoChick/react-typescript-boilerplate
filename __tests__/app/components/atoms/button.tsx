import * as React from 'react'
import { shallow } from 'enzyme'
import * as TestRenderer from 'react-test-renderer'

import Button from '@app/components/atoms/button'

describe('button', () => {
  test('ok', () => {
    const wrapper = shallow(<Button />)
    const buttonWrapper = wrapper.find('button')
    expect(buttonWrapper).toHaveLength(1)
  })
  test('snapshot', () => {
    const tree = TestRenderer.create(<Button />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
