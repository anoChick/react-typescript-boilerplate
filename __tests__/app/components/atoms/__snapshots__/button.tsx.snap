// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`button snapshot 1`] = `
<button>
  Hello!
</button>
`;
